docker/install.sh

```diff
@@ -9,10 +10,17 @@ if [ -e "jenkins_env.sh" ]; then
 	source "jenkins_env.sh"
 fi
 
+# Assumes docker
+docker pull mysql/mysql-server:5.7
+docker run -p 3306:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.7
+
 # catch sbt issues separately
 sbt update && echo "sbt updated successfully."
 
-sbt -J-Xms4g -J-Xmx4g test
+# Assumes docker-machine
+sbt -J-Xms4g -J-Xmx4g test -Ddockermysql.ip=`docker-machine ip default`
```

Comment:
I can see the concern here. There were two approaches in the story, this one (using a dockerized mysql) and setting up a devoted mysql db. I followed this approach as it made more sense since travis supports this, and it's part of normal dev toolkit. There are other benefits like running off the network, outside the vpn, etc. 

Going the other route means setting up a (public?) mysql db that travis could connect to, and that seems to have it's own set of problems like maintenance and access.